packaging/RPMS/Fedora/rabbitmq-server.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=RabbitMQ broker
+After=syslog.target network.target
+
+[Service]
```

Comment:
No, rabbitmq-env.conf is interpreted by rabbitmq scripts themselves, and variables here should be given without `RABBITMQ_` prefix